### ISSUE TITLE
Bad text: All inputs have different type

### ISSUE THREAD

**Issue by jakubklimek** [UnifiedViews/Core#278]
![image](https://cloud.githubusercontent.com/assets/718827/5940435/1609bbbe-a70b-11e4-927f-d790709c0e1e.png)

First of all it should be types, not type.
Second, this happens when a dpu has NO INPUT. This should be handled separately to be more clear.

**Comment by tomas-knap** [comment]
Yes, if the DPU has no input (but it expects some mandatory input), the message must be clear, so that pipeline designer knows input is missing. 

IF there is input, but of wrong type, there should be info saying - "Data unit being mapped to input data unit "input" is of wrong type FILE. Input data unit "input" supports data unit of type "RDF".
